Avoid un-needed downloading od object.infv files, fix conf.py for docs/manual
diff -ur ReviewBoard-1.7.7.1.orig/docs/codebase/conf.py ReviewBoard-1.7.7.1/docs/codebase/conf.py
--- docs/codebase/conf.py	2013-04-22 04:40:29.000000000 +0800
+++ docs/codebase/conf.py	2013-06-16 15:35:59.639256650 +0800
@@ -197,7 +197,3 @@
 
 # If false, no module index is generated.
 #latex_use_modindex = True
-
-
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/dev': None}
diff -ur ReviewBoard-1.7.7.1.orig/docs/manual/conf.py ReviewBoard-1.7.7.1/docs/manual/conf.py
--- docs/manual/conf.py	2013-04-22 04:40:29.000000000 +0800
+++ docs/manual/conf.py	2013-06-16 15:36:30.592256658 +0800
@@ -30,9 +30,14 @@
 
 import reviewboard
 from reviewboard import settings
+from reviewboard.settings import EXTENSIONS_STATIC_ROOT, ROOT_URLCONF
 from django.core.management import setup_environ
+from django import conf
+from settings_local import DATABASES
+conf.EXTENSIONS_STATIC_ROOT=EXTENSIONS_STATIC_ROOT
+conf.settings.DATABASES=DATABASES
 setup_environ(settings)
-
+conf.settings.ROOT_URLCONF=ROOT_URLCONF
 
 # If your extensions are in another directory, add it here. If the directory
 # is relative to the documentation root, use os.path.abspath to make it
@@ -223,7 +223,4 @@
 latex_show_pagerefs = True


-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'http://docs.python.org/dev': None}
-
 todo_include_todos = True
diff -ur 
ReviewBoard-1.7.7.1.orig/docs/releasenotes/conf.py 
ReviewBoard-1.7.7.1/docs/releasenotes/conf.py
--- docs/releasenotes/conf.py	2013-04-22 04:40:29.000000000 +0800
+++ docs/releasenotes/conf.py	2013-06-16 15:36:53.119256665 +0800
@@ -202,7 +202,3 @@
 
 # If false, no module index is generated.
 #latex_use_modindex = True
-
-
-# Example configuration for intersphinx: refer to the Python standard library.
-intersphinx_mapping = {'https://www.reviewboard.org/docs/manual/dev': None}
